libs/search-widget/src/_video-widget/SearchBar.svelte

```diff
@@ -2,7 +2,7 @@
 
 <script lang="ts">
   import type { KBStates } from '@nuclia/core';
-  import { resetStore } from '../core/store';
+  import { resetStore, nucliaStore } from "../core/store";
```

Comment:
prettier got weird, but no need to fix it now

Reply:
I fix it now, otherwise I'll forget